roles/kubernetes-apps/ansible/templates/nodelocaldns-daemonset.yml.j2

```diff
@@ -4,7 +4,8 @@ metadata:
   name: nodelocaldns
   namespace: kube-system
   labels:
-    k8s-app: kube-dns
+    k8s-app: node-local-dns
+    kubernetes.io/cluster-service: "true"
```

Comment:
I think that label is deprecated. (I don't find direct reference, but see kubernetes/kubernetes#72757